projects/desktop/src/components/molecules/singleForm/fields/SFTextField.tsx

```diff
@@ -12,11 +12,12 @@ interface IOwnProps {
     name: string;
     label: string;
     validators: any[];
+    customComponent?: any;
```

Comment:
JSX.Element?

Reply:
It creates a conflict. I guess this is normal to have a any there since there is one is the doc https://github.com/mui-org/material-ui/blob/b73dcb62522795aaca45666623f597200f7f40fb/docs/src/pages/components/text-fields/FormattedInputs.tsx#L96